packages/react-kit/src/lib/config/config.ts

```diff
@@ -10,11 +10,10 @@ if (!magicLinkKey) {
 }
 const infuraKey =
   process.env.STORYBOOK_INFURA_PROJECT_SECRET ||
-  process.env.INFURA_PROJECT_SECRET ||
-  process.env.REACT_APP_INFURA_IPFS_PROJECT_SECRET;
+  process.env.REACT_APP_INFURA_KEY;
 if (!infuraKey) {
   throw new Error(
-    "STORYBOOK_INFURA_PROJECT_SECRET/INFURA_PROJECT_SECRET/REACT_APP_INFURA_IPFS_PROJECT_SECRET is not defined"
+    "STORYBOOK_INFURA_PROJECT_SECRET/REACT_APP_INFURA_KEY is not defined"
```

Comment:
also consistent with https://github.com/bosonprotocol/widgets/blob/a1b60acb8a9b3f423b32cdacbf07d16efdeb900c/.env.example#L52

Reply:
chromatic works (it seems) but double check (there is a GH action)

Reply:
double checked, and fixed !